Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md

```diff
@@ -98,6 +98,24 @@ example.com. IN CAA 0 issuewild "digicert.com"
 
 On the Professional and Enterprise plan, you can manually add your own certificate to your Umbraco Cloud project and bind it to one of the hostnames you've added.
 
+## [Using WAF on Cloud](WAF)
+There is an option to install Web Application Firewall if that is needed.
+If you are using a custom certificate for that specific hostname with WAF make sure you are able to upload the custom certificate:
```

Comment:
Change to "In most cases you need to ensure that the WAF and Umbraco Cloud are using the same certificate on the specific hostname. Custom certificates is a plan-specific feature on Umbraco Cloud, so please make sure that you have access to upload certificates."